### ISSUE TITLE
Applying theme font does not change

### ISSUE THREAD

**Issue by aluciani** [insanum/obsidian_nord#19]
I have a fresh install of Obisdian and when I apply this theme (Obsidian nord) my writing font does not change to reflect what is in the theme (in edit mode) - which looks like Source Code Pro from the screenshot. When I try the Obisdian gruvbox theme all the fonts are changed and work.

Is there something missing that might be causing this?

**Comment by shelbyspees** [comment]
I just ran into this too. I looked at the css in the gruvbox theme and found this snippet setting the CodeMirror font-family:

https://github.com/insanum/obsidian_gruvbox/blob/master/obsidian.css#L261-L267

I isolated it down to this:

```css
.CodeMirror-code {
    font-family: var(--font-monospace);
}
```

@aluciani as a workaround to use the monospace font in the editor for your local theme, create a css file with that snippet under `{vault}/.obsidian/snippets/`.

I'd like to open a PR with the fix as well if I can get to it this week.
